pkg/render/node.go

```diff
@@ -929,6 +929,7 @@ func (c *nodeComponent) nodeVolumeMounts() []v1.VolumeMount {
 		{MountPath: "/var/run/nodeagent", Name: "policysync"},
 		{MountPath: "/typha-ca", Name: "typha-ca", ReadOnly: true},
 		{MountPath: "/felix-certs", Name: "felix-certs", ReadOnly: true},
+		{MountPath: "/host/etc/cni/net.d", Name: "cni-net-dir"},
```

Comment:
Do we want to add this only if Calico CNI is enabled, e.g. https://github.com/tigera/operator/blob/02072527228c00328410e747856922dfce67b845/pkg/render/node.go#L414

Reply:
yeah good call